src/Maker/MakeEntity.php

```diff
@@ -814,4 +880,72 @@ private function getEntityNamespace(): string
     {
         return $this->doctrineHelper->getEntityNamespace();
     }
+
+    private function addEntityFieldWithApiFilters(ClassSourceManipulator $manipulator, array $annotationOptions, string $filter)
+    {
+        $manipulator->addUseStatementIfNecessary('ApiPlatform\\Core\\Annotation\\ApiFilter');
+
+        if ('TermFilter' === $filter || 'MatchFilter' === $filter) {
+            $manipulator->addUseStatementIfNecessary('ApiPlatform\Core\Bridge\Elasticsearch\DataProvider\Filter\\'.$filter);
+        } else {
+            $manipulator->addUseStatementIfNecessary('ApiPlatform\Core\Bridge\Doctrine\Orm\Filter\\'.$filter);
+        }
+
+        $manipulator->addEntityField($filter, $annotationOptions, ['@ApiFilter('.$filter.'::class)']);
+    }
+
+    private function isTypeCompatibleWithApiFilter($data, $apiFilter): bool
+    {
+        $numerics = [
```

Comment:
could be stored in a class constant, same for `$dates`.

Reply:
Done.